rewriter/Minitest.cc

```diff
@@ -891,6 +891,35 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, const ast::
             auto name = makeSharedExamplesConstant(ctx, send->getPosArg(0));
             return ast::MK::Send1(send->loc, move(send->recv), core::Names::include(), send->funLoc, move(name));
         }
+
+        case core::Names::itBehavesLike().rawId(): {
+            if (block != nullptr || !send->recv.isSelfReference() || !insideDescribe || send->numPosArgs() < 1) {
+                return nullptr;
+            }
+
+            // it_behaves_like creates a nested context (describe block) for isolation
```

Comment:
i.e. it's just like `include_context` except it namespaces any lets defined within the shared example